ixwebsocket/IXWebSocket.cpp

```diff
@@ -269,12 +270,11 @@ namespace ix
             if (_stop) return;
 
             // 2. Poll to see if there's any new data available
-            _ws.poll();
-
-            if (_stop) return;
+            WebSocketTransport::PollPostTreatment pollPostTreatment = _ws.poll();
 
```

Comment:
losing this extra _stop early out after calling poll() is a behavior change, but it could be for the best, as now we will make sure that we are doing something after poll() in dispatch.